website/src/app/kb/user-guides/linux-client/readme.mdx

```diff
@@ -1,88 +1,85 @@
 import Alert from "@/components/DocsAlert";
 import SupportOptions from "@/components/SupportOptions";
 
-# Linux Client
+# Linux Headless Client
 
-Firezone supports Linux with a native, headless client for ARMv7l, ARM64, and
-x86-64 architectures.
+Headless Clients are designed to authenticate with long-lived
+[service account](/kb/authenticate/service-accounts) tokens to enable
+system-to-system connectivity where a user isn't present to authenticate with
+your identity provider interactively.
 
-The Linux Client is a lightweight, standalone binary that runs in the background
-using a service account token.
+<Alert color="info">
+  If you're looking for a desktop Client that authenticates with your identity
+  provider instead, see the [Linux GUI
+  Client](/kb/user-guides/linux-gui-client).
+</Alert>
 
 ## Prerequisites
 
 - Any Linux-based OS with kernel 3.10 or higher
-- ARM64, ARMv7l, or x86-64 CPU
-- Administrator access to your Firezone account in order to create a Service
-  Account and generate a token
+- ARM64, ARMv7l, or x86_64 CPU
+- Administrator access to your Firezone account in order to create a service
+  account and generate a token
+- Root access or `sudo` for Split DNS
 
 ## Installation
 
-The Linux Client is currently in beta and can be downloaded from our
-[main repository's releases page](https://github.com/firezone/firezone/releases).
-Alternatively, download the latest Client binary using one of the links below:
+Download the Linux headless Client from our
+[releases page](https://github.com/firezone/firezone/releases),
+or use one of the direct links below:
 
-- [Download the Linux Client for `x86-64`](/dl/firezone-client-headless-linux/latest/x86_64)
+- [Download the Linux Client for `x86_64`](/dl/firezone-client-headless-linux/latest/x86_64)
 - [Download the Linux Client for `ARMv7l`](/dl/firezone-client-headless-linux/latest/armv7)
 - [Download the Linux Client for `ARM64`](/dl/firezone-client-headless-linux/latest/aarch64)
 
+Mark the binary as executable:
+
+```bash
```

Comment:
I know it usually makes sense to add language specifiers to these code blocks, but doing so makes formatting a little ugly since it adds `<code>` tags that don't have the right CSS configured for them. The fix requires some digging into React/react-markdown/hljs and Flowbite's `format` typography plugin.

Opened this to track fix: https://github.com/firezone/firezone/issues/5057

<img width="939" alt="Screenshot 2024-05-20 at 7 12 07 PM" src="https://github.com/firezone/firezone/assets/167144/5209860a-b69a-438e-aac3-2a77bef4d4ab">

Reply:
I'll merge this after that code block fix merges then